requirements-dev-lock.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.7
+# This file is autogenerated by pip-compile with python 3.10
```

Comment:
We need to make sure this lockfile is generated without our lowest supported version.

Reply:
got it i'll revert to what's in `develop` since we aren't adding any new dependencies or updating versions